docs/blog/posts/argocd-keptn-health.md

```diff
@@ -0,0 +1,180 @@
+of a single application are actually working as expected.
+For example, it could be the case that the individual services deployed by ArgoCD are up and
+running, but due to a slow `response time` (let's say `3s`) they are unable to communicate with each other.
+Keptn pre- and post-deployment tasks and evaluations complement the missing functionality
+by providing a straight-forward way to examine the application ability to perform
+actions for which it was developed to do.
+In this particular case, Keptn can perform `KeptnEvaluations` to examine, if the `response time`
+of the application microservices are in the expected boundaries.
+
+<!-- more -->
+
+## How it's going to work?
+
+Keptn and ArgoCD need to be installed and enabled on the same
+cluster.
+To install both components, you can follow the
+[Keptn installation instructions](https://keptn.sh/stable/docs/installation/)
+as well as
+[ArgoCD installation instructions](https://argo-cd.readthedocs.io/en/stable/operator-manual/installation/).
+The reason is that we want to have ArgoCD perform the actual deployment
+of the application and Keptn executing the advanced application health checks.
+
+Additionally, we will need to have an ArgoCD extension, which will consist of
+a React application extending the ArgoCD UI, implemented as
+[ArgoCD UI Application Tab Extension](https://argo-cd.readthedocs.io/en/stable/developer-guide/extensions/ui-extensions/#application-tab-extensions)
+and a proxy allowing Keptn (which will work as a backend service)
+to push the application health status data to the ArgoCD UI.
+For proxy setup, the
+[ArgoCD proxy extension](https://argo-cd.readthedocs.io/en/stable/developer-guide/extensions/proxy-extensions/)
+will be used.
+
+## What's the added value of Keptn?
+
+Let's try to show a real-life example of an application deployed via ArgoCD,
+which has a healthy green status in ArgoCD UI, but it's not working as expected
+due to a slow `response time` of the application.
+
+We will deploy a simple `podtato-head` application, which consists of multiple
+Deployments and Services via ArgoCD.
+The Argo Application deploying the manifests can look like the following:
+
+```yaml
+{% include "./argocd-keptn-health/argo-app.yaml" %}
+```
+
+After a few moments, the `podtato-head` application is successfully deployed and all pods
+are running.
+
+![Running Pods](./argocd-keptn-health/running-pods.png)
+
+We can also examine the ArgoCD UI and everything seems to be working as expected and the
+`podtato-head` application is healthy.
+
+![Healthy App](./argocd-keptn-health/healthy-app.png)
+
+Let's now try to add some health checks of the `podtato-head` application
+and use Keptn to execute them.
+For this, we are going to use the
+[Keptn Release Lifecycle Management](https://keptn.sh/stable/docs/getting-started/lifecycle-management/)
+feature and perform the checks via `KeptnEvaluations`.
+For this example, we assume that you already have a metrics provider deployed
```

Comment:
Do you want to reference https://keptn.sh/stable/docs/guides/evaluatemetrics/#define-keptnmetricsprovider-resources for instructions about how to do this?

Reply:
here we mean more metrics provider like Prometheus, Datadog or Dynatrace